build/common.sh

```diff
@@ -45,7 +45,7 @@ readonly KUBE_GCS_DELETE_EXISTING="${KUBE_GCS_DELETE_EXISTING:-n}"
 
 # Constants
 readonly KUBE_BUILD_IMAGE_REPO=kube-build
-readonly KUBE_BUILD_IMAGE_CROSS_TAG="v1.6.2-2"
+readonly KUBE_BUILD_IMAGE_CROSS_TAG="$(cat build/build-image/cross/VERSION)"
```

Comment:
Oh, woops. the Makefile automatically pushes.  I guess it is pushed now.
Sigh.

On Mon, Jul 11, 2016 at 10:12 PM, Lucas Käldström <notifications@github.com>
wrote:

> In build/common.sh
> <https://github.com/kubernetes/kubernetes/pull/28819#discussion_r70377957>
> :
>
> > @@ -45,7 +45,7 @@ readonly KUBE_GCS_DELETE_EXISTING="${KUBE_GCS_DELETE_EXISTING:-n}"
> >
> >  # Constants
> >  readonly KUBE_BUILD_IMAGE_REPO=kube-build
> > -readonly KUBE_BUILD_IMAGE_CROSS_TAG="v1.6.2-2"
> > +readonly KUBE_BUILD_IMAGE_CROSS_TAG="$(cat build/build-image/cross/VERSION)"
>
> We've debated about this earlier, but I think I'm fine with having this.
> The pro is no manual update here again, the con is that we must always
> have the stable kube-cross image in here. But that don't bother me much.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/28819/files/1aab7ca76947d57368f386cf668f9d428cb88f79#r70377957>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFVgVHQ2y0sQwHa1vec4fiPR8jZEloraks5qUyJGgaJpZM4JKCh5>
> .
>